samples/40.timex-resolution/src/pom.xml

```
@@ -0,0 +1,248 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.microsoft.bot.sample</groupId>
+  <artifactId>timex-resolution</artifactId>
```

This can be: 
```suggestion
  <artifactId>bot-timexresolution</artifactId>
```

or
```suggestion
  <artifactId>bot-timex-resolution</artifactId>
```